docs/root/version_history/current.rst

```diff
@@ -30,6 +30,7 @@ Bug Fixes
 * csrf: fixed issues with regards to origin and host header parsing.
 * dynamic_forward_proxy: only perform DNS lookups for routes to Dynamic Forward Proxy clusters since other cluster types handle DNS lookup themselves.
 * fault: fixed an issue with `active_faults` gauge not being decremented for when abort faults were injected.
+* rocketmq_proxy network-level filter: fixed a crash involving incorrect header lengths.
```

Comment:
Thanks! Can we make "crash" more clear? Something like "debug mode crash and release mode underflow"?

Reply:
> Thanks! Can we make "crash" more clear? Something like "debug mode crash and release mode underflow"?

Added! Thanks!